src/distilabel/steps/task/quality_scorer.py

```diff
@@ -75,21 +77,22 @@ def format_input(self, input: str) -> ChatType:  # type: ignore
     @property
     def outputs(self):
         """The output for the task is the `generation` and the `model_name`."""
-        return ["quality_score"]
+        return ["quality_scores"]
```

Comment:
Maybe we can just leave it as `scores`? Note that the user will always have the possibility to rename it to `quality_scores` or any other name if `scores` is confusing, while keeping `scores` for the most common scenarios 👍🏻 

```suggestion
        return ["scores"]
```